lib/ansible/module_utils/network/aci/aci.py

```diff
@@ -483,7 +485,7 @@ def _construct_url_4(self, root, sec, parent, obj, child_includes):
         obj_class = obj['aci_class']
         obj_rn = obj['aci_rn']
         # obj_filter = obj['filter_target']
```

Comment:
I know it was here before, though why do we have commented out code?

Reply:
I need to update this method to provide more granular facts data like the previous methods. I started working on this, but I haven't had time to test the API for all scenarios and see what kind of data is returned. These will either be uncommented or removed once I get around to fleshing out this method.